variants/web/package.json.rb

```diff
@@ -0,0 +1,22 @@
+insert_into_file 'package.json', after: %r{"private":.+\n} do
+  <<~EOT
+    "scripts": {
+      "lint": "eslint . --color",
+      "lint:fix": "eslint . --color --fix"
+    },
+  EOT
+end
+
+insert_into_file 'package.json', after: %r{"@rails/ujs":.+\n} do
+  <<~EOT
+    "i18n-js": "^3.0.11",
```

Comment:
Use a fixed version: "3.0.11" (no ^)

Reply:
Also what about updating to [3.6.0](https://github.com/fnando/i18n-js/releases/tag/v3.6.0)?

Reply:
Update and lock the version on the Gemfile as well 👍 - https://github.com/nimblehq/rails-templates/pull/156/commits/f466ec1fee7faf997d161e515eb0774b10df7c36